### ISSUE TITLE
[Visual requirement – Make Code – Home Page] Luminosity ratio is less than 3:1 for focus indicator present on all tiles under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3438]
**User Experience:**
Visual impaired user will find difficulty to see the focus indicator over the control or element if that focus indicator have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify contrast ratio of focus indicator over the all titles control like(New Project, Flashing Heart, Name tag, Smiley Buttons, Dice etc.)

**Actual Result:**
Luminosity ratio is less than 3:1 for focus indicator over the all titles control like(New Project, Flashing Heart, Name tag, Smiley Buttons, Dice etc.) under ‘Home page’ and due to this, visual impaired user is unable to see the focus indicator. 

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for focus indicator over the all titles control like(New Project, Flashing Heart, Name tag, Smiley Buttons, Dice etc.) under ‘Home page’, so that visual impaired user could easily see the border of focus. Same issue exists on 'ARM template & Close' controls on same blade.

**Note:**
Same issue exists on Firefox & Chrome.

**MAS Reference:**
[MAS 1.4.11 - Non-text Contrast
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=f2aa2853-82ab-42f5-877d-afa7e900c553)

**Reference Links:**
•	[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA
#URL:https://makecode.microbit.org/#
